lightspeed/modules/proc_remove-seat-licenses.adoc

```diff
@@ -5,10 +5,14 @@
 = Removing seat licenses
 
 [role="_abstract"]
-You can remove users who no longer need to access {LightspeedShortName}, which makes licenses available for others to use.
+You can remove users who no longer need to access {LightspeedShortName}, which makes licenses available for others to use. 
+
+You can also remove seat licenses of deactivated RH-SSO users, so that they can no longer access {LightspeedShortName}. When RH-SSO users are deactivated, their {LightspeedShortName} seat licenses are not removed automatically. These users can continue accessing {LightspeedShortName} until the next time they are forced to reauthenticate with RH-SSO. To prevent deactivated RH-SSO users from accessing {LightspeedShortName}, you must remove their {LightspeedShortName} seat licenses.
```

Comment:
This looks good but I wonder if we should say "You should" instead of "You can".

Reply:
Thank you for the explanation. I still can't get comfortable with the "You can" in the first sentence. Given the sentence before it, the first half of the sentence feels like we're offering up another way to get more licenses freed up for other users, and if I don't care about that at the moment, I will stop reading. Maybe we should just delete the first sentence entirely and replace it with "IMPORTANT: " or something?

Reply:
I understand your point, and have made minor edits. Could you please check if the current wording works? Thank you.

Reply:
Incorporated additional updates per Slack convo., thank you.